src/modules/curations/itemCuration/reducer.ts

```diff
@@ -65,7 +67,19 @@ export function itemCurationReducer(state: ItemCurationState = INITIAL_STATE, ac
         ...state,
         loading: loadingReducer(state.loading, action)
       }
+    case FINISH_TP_APPROVAL_FLOW: {
+      const { collection, itemCurations } = action.payload
 
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        data: {
+          ...state.data,
+          [collection.id]: [...itemCurations]
+        },
+        error: null
```

Comment:
The reducer clears the errors and the loading action, but this won't be necessary, as the loading reducer won't work (because of the name of the action) and I don't know about the errors, but I think that they shouldn't be cleared either, right?

Reply:
You're right! got confused with the entire approval flow